### ISSUE TITLE
Voiko puhua tekstien sekventiaalisuudesta jo teoriaosassa?

### ISSUE THREAD

**Issue by hrmJ** [hrmJ/100-tapaa-indikoida-topiikkia#15]

**Comment by Kirde80** [comment]
Moro! Täälläpä ei ole eduroamia, joten pääsen vasta illemmalla kissaohjelmaan. Jos lähetän sulle aamuksi vielä viimeisen version silmäilyyn, kunhan illalla fiksaan loppuun?
Kirsi


Lähetetty Samsung Galaxy -älypuhelimesta.
-------- Alkuperäinen viesti --------Lähettäjä: Juho Härme <notifications@github.com> Päivämäärä: 7.2.2018  14.46  (GMT+01:00) Saaja: hrmJ/100-tapaa-indikoida-topiikkia <100-tapaa-indikoida-topiikkia@noreply.github.com> Kopio: Kirde80 <kirsi.elomaki@gmail.com>, Assign <assign@noreply.github.com> Aihe: Re: [hrmJ/100-tapaa-indikoida-topiikkia] Voiko puhua tekstien sekventiaalisuudesta jo teoriaosassa? (#15) 
Closed #15.

—
You are receiving this because you were assigned.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/hrmJ/100-tapaa-indikoida-topiikkia","title":"hrmJ/100-tapaa-indikoida-topiikkia","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/hrmJ/100-tapaa-indikoida-topiikkia"}},"updates":{"snippets":[{"icon":"DESCRIPTION","message":"Closed #15."}],"action":{"name":"View Issue","url":"https://github.com/hrmJ/100-tapaa-indikoida-topiikkia/issues/15#event-1462389845"}}}

**Comment by hrmJ** [comment]
Joo, siis saitko sen mun uusimman version kuitenkin? Saahan sen selaimenki  kautta ladattua ja printattua

**Comment by Kirde80** [comment]
Juu sain uusimman, pääsin lopulta edu-verkkoonkin. Luin tekstin kerran läpi
ja näytti hyvältä. Täydennän omalta osaltani puuttuvat tiedot illalla
hotellilla ja saatan hiukan viilata pilkkua :D Laitan tätä kautta viestin,
kun siirrän oman versioni vielä sulle tarkistettavaksi.



2018-02-08 15:25 GMT+02:00 Juho Härme <notifications@github.com>:

**Comment by Kirde80** [comment]
Huomenta! Mä silmäilen tässä juuri vielä kerran, mitä eilen illalla tein ja
pushaan sitten tuotuoksen. Täällä on kello tunnin vähemmän, joten mulla on
vielä kans jonkun verran aamuapalan jälkeen aikaa säätää, ennen kuin täytyy
lähteä yliopistolle.

Kirsi

2018-01-10 9:45 GMT+02:00 Juho Härme <notifications@github.com>:
